autogpt_platform/frontend/src/app/(platform)/build/components/FlowEditor/Flow/useFlow.ts

```diff
@@ -144,5 +151,41 @@ export const useFlow = () => {
     };
   }, []);
 
-  return { isFlowContentLoading: isGraphLoading || isBlocksLoading };
+  // Drag and drop block from block menu
+  const onDragOver = useCallback((event: React.DragEvent) => {
+    event.preventDefault();
+    event.dataTransfer.dropEffect = "copy";
+  }, []);
+
+  const onDrop = useCallback(
```

Comment:
We shouldn't need `useCallback` here unless we see performance issues? The drop event should only happens once or twice so I think no need to over-optimise for now 🙇🏽